src/NServiceBus.Core/Unicast/BackwardCompatibility/MutateMessageContentTypeOfIncomingTransportMessages.cs

```diff
@@ -15,15 +14,12 @@ class MutateMessageContentTypeOfIncomingTransportMessages : IMutateIncomingTrans
         /// <param name="transportMessage">Transport Message to mutate.</param>
         public void MutateIncoming(TransportMessage transportMessage)
         {
-            if (!TransportMessageExtensions.IsControlMessage(transportMessage.Headers) && !transportMessage.Headers.ContainsKey(Headers.ContentType))
-            {
-                transportMessage.Headers[Headers.ContentType] = Serializer.ContentType;
-            }
+            
```

Comment:
@SimonCropp @andreasohlund I asked @Particular/engineers if it was ok to remove, since the type is Obsoleted in v6. They said 'yip'